### PR TITLE
OpenGov Treasury Doc

### LINKED ISSUE #4473: Kusama Treasury doc - OpenGov update
The treasury page of Kusama guide needs an update. Add all the treasury-related tracks and their params/info.

https://guide.kusama.network/docs/learn-treasury/

### PR DESCRIPTION
**Pull request by DrW3RK**
Work in progress

### CODE REVIEW

**Comment by DrW3RK** [comment]
@filippoweb3 Adds a basic treasury page to Polkadot Wiki and Kusama guide. Things to do with other PRs-

1. Move current treasury docs to Archive
2. Add a note on current treasury docs to use Polkadot OpenGov for accessing Treasury funds
3. Refine Polkadot OpenGov Treasury doc

**Review by filippoweb3 — APPROVED** [review]
lgtm, added your requests @DrW3RK and I added a bit more content about treasury inflows and outflows. let me know what you think!

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+- **Transaction fees:** 80% of the transaction fees of every submitted extrinsic is diverted to the
+  Treasury, while 20% is given to the block producers.
+- **Staking inefficiencies:** the network knows an exogenously determined parameter called ideal
+  staking rate. The APY for stakers (nominators & validators) decreases whenever the actual staking
+  rate is not equal to the ideal staking rate. To keep inflation constant at 10%, the system does
+  not creates less tokens, rather some share of the overall reward for stakers is diverted to the
+  Treasury (more information
+  [here](https://research.web3.foundation/Polkadot/overview/token-economics)).
+- **Slashes:** whenever validators and nominators are slashed, a share of the slashed tokens are
+  diverted to Treasury. This is a rare and unpredictable event.
```
Comment (profradha) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
  diverted to Treasury. They are typically rare and unpredictable events.
```

File: docs/learn/learn-polkadot-opengov-treasury.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+- **Transfers:** everyone can send funds to the Treasury directly. This is a rare event and mostly
```
Comment (profradha) [inline docs/learn/learn-polkadot-opengov-treasury.md]:
```suggestion
- **Transfers:** everyone can send funds to the Treasury directly. This is a rare event and typically
```

**Pull request merged**